pkg/model/mongodb_doc.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model
+
+import "go.mongodb.org/mongo-driver/bson/primitive"
+
+type LabelDoc struct {
+	ID       primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
+	Labels   map[string]string  `json:"labels,omitempty"`
+	Revision int                `json:"revision"`
+	Domain   string             `json:"domain,omitempty"` //tenant info
+}
+type KVDoc struct {
+	ID        primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
+	LabelID   string             `json:"label_id,omitempty" bson:"label_id,omitempty"`
+	Key       string             `json:"key"`
+	Value     string             `json:"value,omitempty"`
+	ValueType string             `json:"value_type,omitempty" bson:"value_type,omitempty"` //ini,json,text,yaml,properties
+	Checker   string             `json:"check,omitempty"`                                  //python script
+
+	Labels   map[string]string `json:"labels,omitempty"` //redundant
+	Domain   string            `json:"domain,omitempty"` //redundant
```

Comment:
Can we treat the Domain string as a Label?

Reply:
it is tenant, we can not let user decide multi-tenant